src/mgmt/services/manager/project/api/settings.py

```diff
@@ -3,7 +3,24 @@
 #
 # Authors: Bin Liang <@liangbin>
 #
+from kubernetes.client.rest import ApiException
+from kubernetes import client, config
+import logging
+logger = logging.getLogger()
+
+
+config.load_incluster_config()
+core_api = client.CoreV1Api()
 
 def init():
     global activeZgc
     activeZgc = {}
+    global zgc_cidr_range
+    # Give it a default value.
+    zgc_cidr_range = '10.0.0.0/16'
+    try:
+        api_response = core_api.list_config_map_for_all_namespaces(label_selector='name=zgc-cluster-config')
+        logger.info('Result for getting configmap: {}'.format(api_response))
+        zgc_cidr_range = api_response.items[0].data['cidr']
+    except ApiException as e:
+        logger.error('Exception when updating exsiting droplet: {}'.format(e))
```

Comment:
The error message is not correctly indicate when it happened, also add newline at end of file.
Add MAC_PRIVATE = 0x828300000000 to config map as well

Reply:
fixed